src/main/resources/liquibase/202308211518-rename-rhel-product-ids.xml

```diff
@@ -0,0 +1,43 @@
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="202308211518" author="lburnett">
+    <comment>Consider "RHEL Server" as "RHEL for x86" product id for snapshots.</comment>
```

Comment:
@lindseyburnett If I understand this correctly, here we are migrating `RHEL server` snapshots and considering it as `RHEL for x86`. This will work correctly for stage, but this will do migration in production env as well, where we have both `RHEL server` and `RHEL for x86` snapshots and those won't be the same snapshots. Am I understanding it correctly ?

Reply:
This is resolved in slack conversation. The summary is we are redefining RHEL for x86 to RHEL Server with x86